packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Understand MFA enrollment policy API changes after the upgrade
+title: Understand MFA and authenticator enrollment policy API changes after the upgrade
```

Comment:
should it be just `Understand authenticator enrollment policy API changes after the upgrade`?